api/src/models/pitch.ts

```diff
@@ -0,0 +1,105 @@
+import mongoose, { Document, Schema } from 'mongoose';
+import { IUser } from './user';
+
+export interface IPitch extends Document<any> {
+  name: string;
+  pitchAuthor: mongoose.Types.ObjectId;
+  pitchStatus: string;
+  assignmentStatus: string;
+  assignmentGoogleDocLink: string;
+  assignmentContributors: [IUser];
+  topic: string;
+  currentWriters: number;
+  targetWriters: number;
+  currentEditors: number;
+  targetEditors: number;
+  currentData: number;
+  targetData: number;
+  currentVisuals: number;
+  targetVisuals: number;
+  currentIllustration: number;
+  targetIllustration: number;
+  currentPhotography: number;
+  targetPhotography: number;
+  currentFactChecking: number;
+  targetFactChecking: number;
+  currentRadio: number;
+  targetRadio: number;
+  currentLayout: number;
+  targetLayout: number;
+  approvedBy: IUser;
+  similarStories: [string];
+  deadline: Date;
+}
+
+const pitchStatusEnum = {
+  APPROVED: 'APPROVED',
+  PENDING: 'PENDING',
+  REJECTED: 'REJECTED',
+  NONE: 'NONE',
+};
+
+const assignmentStatusEnum = {
+  IN_PROGRESS: 'IN_PROGRESS',
+  COMPLETED: 'COMPLETED',
+  ABANDONED: 'ABANDONED',
+  NONE: 'NONE',
+};
+
+// TODO: don't repeat this code
```

Comment:
Are you planning on extracting this enum to another file? Maybe we can make a file that holds all the enums and import from there in case we need to reuse more of them later on

Reply:
oh yes that was my idea because both the User model and Pitch model uses some of the same enums